pkg/model/manifest_test.go

```diff
@@ -155,6 +155,7 @@ func Test_validateDivert(t *testing.T) {
 }
 
 func TestInferFromStack(t *testing.T) {
+	dirtest := "/stack/dir/"
```

Comment:
This might have issues with other OS, can we use something like `filepath` or `os`?

Reply:
`filepath.IsAbs` behaviour is different on linux and windows (on windows it expects the disk first. for example `C:/` and I don't know if it can affect) using `filepath.Clean("/stack/dir/")` would do the trick